Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,95 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the release documentation, the Kata Containers project makes use of
+semantic versioning.  A release version is described by MAJOR.MINOR.PATCH.  Early in
+the project, we will introduce many new features which will require a minor version change
+as well as bug fixes.  To facilitate a stable user environment, Kata will begin to provide
+stable branch based releases as well as a master branch release.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created off of master, a new stable branch will be created for
+the prior MAJOR or MINOR release and the older stable branch will no longer be maintained.
```

Comment:
What happens to that branch?:

- Is it deleted or somehow marked "unmaintained"?
- What about references to the branch "elsewhere"?
- What about OBS packages / branches for this branch release?
- What about users running on that release? We need to explain how they can:
  - upgrade the the next stable release
  - stay on that release (but with big fat warnings about (lack of) security updates, etc).